scattering/scattering.py

```diff
@@ -14,7 +14,7 @@
 
 
 
-def structure_factor(trj, Q_range=(0.5, 50), n_points=1000, framewise_rdf=False):
+def structure_factor(trj, pair = None, Q_range=(0.5, 50), n_points=1000, framewise_rdf=False):
```

Comment:
Please see PEP8 style guide.  Arguments in a function should be formatted like this: `pair=None`

Reply:
https://www.python.org/dev/peps/pep-0008/